tests/test_frozenbot.py

```diff
@@ -66,15 +66,15 @@ def test3():
 
     assert {cmd.name for cmd in bot.available_commands()} == {
```

Comment:
As I said before, this should be a list: you aren't checking the ordering of the elements :)